operations/security_and_authentication/Troubleshoot_Kyverno_Configuration_manually.md

```diff
@@ -0,0 +1,25 @@
+# How to troubleshoot Kyverno configuration manually
+
+## Check if Kyverno pods are up and running
+
+Run the script with -p option.
+
+```bash
+./k8s_kyverno_pods_running.sh -p
```

Comment:
Hi Pradeep - where would the user get the script if they wanted to use it for troubleshooting - where does it get installed on the system?   We should show the whole pathway.

Reply:
I think we want to give them the pathway to find these scripts so that if they need to run them, they are not searching for them on the system..  I dropped the path in the slack channel.